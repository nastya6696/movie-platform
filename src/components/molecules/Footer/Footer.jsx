import styles from './styles.module.scss';
import {LogoIcon} from "../../atoms";

export const Footer = () => (
  <div className={styles.footer}>
    <LogoIcon boldPart="netflix" thinPart="roulette"/>
  </div>
)
