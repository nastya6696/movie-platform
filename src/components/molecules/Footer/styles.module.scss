@import "../../../assets/scss/common";

.footer {
  background-color: $grey;
  height: 70px;
  width: 100%;
  padding: 23px 525px;
  text-align: center;
}
