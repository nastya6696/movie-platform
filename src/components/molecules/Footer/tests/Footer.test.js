import {Footer} from "../Footer";
import {create} from 'react-test-renderer';

jest.mock('../../../atoms/LogoIcon', () => ({
  LogoIcon: (({ boldPart, thinPart }) => `
     (--- LogoIcon ---): props {
     boldPart: '${boldPart}',
     thinPart: '${thinPart}'
    }`
  )
}));

describe('Footer', () => {
  it('should create simple Footer component', () => {
    const component = create(<Footer />);

    expect(component).toMatchSnapshot();
  })
});
