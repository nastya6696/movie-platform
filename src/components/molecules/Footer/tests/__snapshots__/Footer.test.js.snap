// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should create simple Footer component 1`] = `
<div
  className="footer"
>
  
     (--- LogoIcon ---): props {
     boldPart: 'netflix',
     thinPart: 'roulette'
    }
</div>
`;
